components/multiselect/custom-value.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Custom Value
+page_title: MultiSelect - Custom Value
+description: Custom values and user input in the MultiColumnComboBox for Blazor.
+slug: multiselect-custom-value
+tags: telerik,blazor,multiselect,custom,value,input
+published: True
+position: 16
+---
+
+# MultiSelect Custom Values
+
+The MultiSelect component allows the user to type in their own value that is not a part of the predefined set of options that the developer provided.
+
+The text entered by the user can still go into the field the combo box is bound to through two-way binding.
```

Comment:
Is combo box correct here? (Just checking if it's a leftover from MultiColumnComboBox.)